data/SRP018247.lst

```diff
@@ -1,3 +1,6 @@
+SRR655969
```

Comment:
Why did you include these three samples in the list? They're excluded from the analysis, so it's not clear why we'd need to have them downloaded, etc.

Reply:
OK, I see what happened: the numbers aren't sorted from the original sheet, and you've added them in duplicate. I'll take care of this.